src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/NamedTypeSymbolReferenceFinder.cs

```diff
@@ -115,31 +118,34 @@ protected override Task<ImmutableArray<string>> DetermineGlobalAliasesAsync(INam
         // First find all references to this type, either with it's actual name, or through potential
         // global alises to it.
         await AddReferencesToTypeOrGlobalAliasToItAsync(
-            namedType, state, initialReferences, cancellationToken).ConfigureAwait(false);
+            namedType, state, StandardCallbacks<FinderLocation>.AddToArrayBuilder, initialReferences, cancellationToken).ConfigureAwait(false);
+
+        // call processResult on items in initialReferences
```

Comment:
comment needs fleshing out.  maybe like: these shoudl both be reported as normal results, but are also used later on to findmore results.